example/lib/src/screen/home_page.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/material.dart';
+import '../models/widget_route.dart';
+import '../resources/route_constants.dart';
+import '../utils/ui_constants.dart';
+
+class HomePage extends StatefulWidget {
+  HomePage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  late List<WidgetRoute> _widgetList;
+
+  List<WidgetRoute> _generateRoutes() {
+    List<WidgetRoute> _routeList = [];
+    _routeList.add(
+      WidgetRoute(
+        title: UiConstants.customAppBarTitle,
+        widgetRoute: customAppBar,
+      ),
+    );
+    _routeList.add(
+      WidgetRoute(
+        title: UiConstants.movieDescriptionTitle,
+        widgetRoute: movieDescription,
+      ),
+    );
+
+    _routeList.add(
+      WidgetRoute(
+        title: UiConstants.movieMainInfoTitle,
+        widgetRoute: movieMainInfo,
+      ),
+    );
+
+    _routeList.add(
+      WidgetRoute(
+        title: UiConstants.movieDetailsImageTitle,
+        widgetRoute: movieDetailsImage,
+      ),
+    );
+
+    return _routeList;
+  }
+
+  @override
+  void initState() {
+    super.initState();
+    _widgetList = _generateRoutes();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Text(
+              UiConstants.appBarTitle,
+            ),
+          ],
+        ),
+      ),
```

Comment:
why do you have a column with just one child? you should use the appBar property centerTitle: true, to center the title, and not align it with a column

Reply:
Column removed.